src/components/btn_event_feedback_link.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+
+
+const FeedbackLink = ({link}) => (
+  <div className="float">
+    <a href={link} 
```

Comment:
Somehow this doesn't get rendered properly, the rendered button on the page doesn't seem to have the `href` attribute, thus it's not clickable. I think you might wanna investigate this further.